tests/ha/priority_fencing_delay.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test the priority fencing delay feature
+# The node with the master resource must always win the fencing match
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+
+sub stonith_iptables {
+    my ($self, $count, $cluster) = @_;
+    my $partner_ip = is_node(1) ? get_ip(choose_node(2)) : get_ip(choose_node(1));
+
+    while ($count ne 0) {
```

Comment:
Better to use `!=` rather than `ne` in Perl for numeric test I think.

Reply:
sure... the same mistake as in the preflight check PR, at least, I'm consistent...

Reply:
Actually, you could simply do `while ($count) {`.

When the count reaches 0, it will evaluate to false and finish the loop.

Reply:
BTW, you will probably will need to check that `$count` is greater than zero before starting the loop. Or it will loop more times than you intend :)